doc/binary_format.md

```diff
@@ -54,7 +54,7 @@ This input enumerates all factors in the factor graph that refers to variables a
 The following fields are repeated for each factor.
 
     factorFunction  uint16_t    2   // type of factor function, see: FACTOR_FUNCTION_TYPE
-    arity           uint64_t    8   // arity of the factor, i.e., how many variables it connects to
+    arity           uint32_t    4   // arity of the factor, i.e., how many variables it connects to
```

Comment:
I thought 64 was better as there could be factors that connect to most of the of variables